### PR TITLE
KFSPTS-12308: Changes needed to keep existing PMW code working due to PMW non-unique tax id number change.

### PR DESCRIPTION
**Pull request by nkimble**

### CODE REVIEW

**Comment by nkimble** [comment]
This code PR depends upon SQL PR#371 and they must be applied together

File: src/main/java/edu/cornell/kfs/pmw/batch/PaymentWorksParameterConstants.java [hunk 1]
```diff
@@ -4,15 +4,15 @@
...
-    public static final String PAYMENTWORKS_NEW_VENDOR_REPORT_PENDING_NEW_VENDORS_SUCCESSFULLY_PROCESSED_LABEL = "PAYMENTWORKS_NEW_VENDOR_REPORT_PENDING_NEW_VENDORS_SUCCESSFULLY_PROCESSED_LABEL";
-    public static final String PAYMENTWORKS_NEW_VENDOR_REPORT_PENDING_NEW_VENDORS_ERRORED_LABEL = "PAYMENTWORKS_NEW_VENDOR_REPORT_PENDING_NEW_VENDORS_ERRORED_LABEL";
+    public static final String PAYMENTWORKS_NEW_VENDOR_REPORT_APPROVED_NEW_VENDORS_FOUND_LABEL = "PAYMENTWORKS_NEW_VENDOR_REPORT_APPROVED_NEW_VENDORS_FOUND_LABEL";
+    public static final String PAYMENTWORKS_NEW_VENDOR_REPORT_APPROVED_NEW_VENDORS_NOT_PROCESSED_LABEL = "PAYMENTWORKS_NEW_VENDOR_REPORT_APPROVED_NEW_VENDORS_NOT_PROCESSED_LABEL";
+    public static final String PAYMENTWORKS_NEW_VENDOR_REPORT_APPROVED_NEW_VENDORS_SUCCESSFULLY_PROCESSED_LABEL = "PAYMENTWORKS_NEW_VENDOR_REPORT_APPROVED_NEW_VENDORS_SUCCESSFULLY_PROCESSED_LABEL";
+    public static final String PAYMENTWORKS_NEW_VENDOR_REPORT_APPORVED_NEW_VENDORS_ERRORED_LABEL = "PAYMENTWORKS_NEW_VENDOR_REPORT_APPROVED_NEW_VENDORS_ERRORED_LABEL";
```
Comment (cah292) [inline src/main/java/edu/cornell/kfs/pmw/batch/PaymentWorksParameterConstants.java]: Minor typo in name of constant.

File: src/main/java/edu/cornell/kfs/vnd/businessobject/VendorDetailExtension.java [hunk 2]
```diff
@@ -37,6 +38,9 @@
     protected List<CuVendorCreditCardMerchant> vendorCreditCardMerchants;
     protected String procurementMethods;
 
+    protected boolean paymentWorksOriginatingIndicator;
+    protected Timestamp paymentWorksLastActivityTimestamp;
```
Comment (cah292) [inline src/main/java/edu/cornell/kfs/vnd/businessobject/VendorDetailExtension.java]: The new PR code is populating these two new properties accordingly, but they do not appear to have been added to the UI by this new code. Are these two properties meant to be displayed on the KFS screens and documents? If so, should that setup be done in this PR or in a future PR?

File: src/main/java/edu/cornell/kfs/pmw/batch/PaymentWorksParameterConstants.java (hunk 1 above)
Comment (nkimble) [reply src/main/java/edu/cornell/kfs/pmw/batch/PaymentWorksParameterConstants.java]: fixed

File: src/main/java/edu/cornell/kfs/vnd/businessobject/VendorDetailExtension.java (hunk 2 above)
Comment (nkimble) [reply src/main/java/edu/cornell/kfs/vnd/businessobject/VendorDetailExtension.java]: Cathy indicated that she wanted these new attributes in place so the source of the vendor data could be tracked and reported on.  She did not specify these attributes should be added to the UI. If Cathy or purchasing decides that these items are needed on the UI, they could be added as we will accurately be capturing the data from the beginning of the PMW implementation.

**Pull request merged**